node/pkg/processor/observation.go

```diff
@@ -247,36 +247,8 @@ func (p *Processor) handleInboundSignedVAAWithQuorum(ctx context.Context, m *gos
 		return
 	}
 
-	// Check if VAA doesn't have any signatures
-	if len(v.Signatures) == 0 {
-		p.logger.Warn("received SignedVAAWithQuorum message with no VAA signatures",
-			zap.String("digest", hash),
-			zap.Any("message", m),
-			zap.Any("vaa", v),
-		)
-		return
-	}
-
-	// Verify VAA has enough signatures for quorum
-	quorum := CalculateQuorum(len(p.gs.Keys))
-	if len(v.Signatures) < quorum {
-		p.logger.Warn("received SignedVAAWithQuorum message without quorum",
-			zap.String("digest", hash),
-			zap.Any("message", m),
-			zap.Any("vaa", v),
-			zap.Int("wanted_sigs", quorum),
-			zap.Int("got_sigs", len(v.Signatures)),
-		)
-		return
-	}
-
-	// Verify VAA signatures to prevent a DoS attack on our local store.
-	if !v.VerifySignatures(p.gs.Keys) {
-		p.logger.Warn("received SignedVAAWithQuorum message with invalid VAA signatures",
-			zap.String("digest", hash),
-			zap.Any("message", m),
-			zap.Any("vaa", v),
-		)
+	if err := v.Verify(p.gs.Keys); err != nil {
+		p.logger.Warn("dropping SignedVAAWithQuorum message because it failed verification: " + err.Error())
```

Comment:
This is fine too but there is a [`zap.Error()`](https://pkg.go.dev/go.uber.org/zap#Error) function for logging errors.

Reply:
I couldn't get that to work and it was trying to import a new zap package.  So, I left it out for simplicity.